lib/Listener/MessageCacheUpdaterListener.php

```diff
@@ -60,10 +60,14 @@ public function handle(Event $event): void {
 
 			$this->mapper->update($message);
 		} elseif ($event instanceof MessageDeletedEvent) {
```

Comment:
Every dispatch of MessageDeletedEvent should be updated to pass an array of uids. How about renaming it to MessagesDeletedEvent?